### PR TITLE
added test for msg_server_buy_storage

### PR DESCRIPTION
**Pull request by nuggetnchill**
👋 go test -v -run TestStorageTestSuite/TestBuyStorage

### CODE REVIEW

**Comment by BiPhan4** [comment]
a couple checks missing from for loop. Already discussed in PM.

File: x/storage/keeper/msg_server_buy_storage_test.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+		expErrMsg string
+	}{
+		{
+			testName: "buy 1gb for 2 month",
+			msg: types.MsgBuyStorage{
+				Creator:      testAccount.String(),
+				ForAddress:   "testAccount.String()",
+				Duration:     "432000",
+				Bytes:        "2000000000",
+				PaymentDenom: "ujkl",
```
Comment (TheMarstonConnell) [inline x/storage/keeper/msg_server_buy_storage_test.go:32]: isn't this 2gb?

File: x/storage/keeper/msg_server_buy_storage_test.go [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+				Duration:     "432000",
+				Bytes:        "1000000000",
+				PaymentDenom: "uatom",
+			},
+			expErr:    true,
+			expErrMsg: "cannot pay with anything other than ujkl: invalid coins",
+		},
+		{
+			testName: "invalid address",
+			msg: types.MsgBuyStorage{
```
Comment (TheMarstonConnell) [inline x/storage/keeper/msg_server_buy_storage_test.go:75]: One final test case to have valid sender & invalid for address, and then one with invalid sender and valid for address and then I'd say this is perfect :)

**Review by TheMarstonConnell — CHANGES_REQUESTED** [review]

**Comment by nuggetnchill** [comment]
@TheMarstonConnell good thing u pointed out to check both creator and for address, please take a look again 😄

**Pull request merged**